frontend/src/component/demo/demo-topics.tsx

```diff
@@ -467,6 +479,17 @@ export const TOPICS: ITutorialTopic[] = [
                 ),
                 preventDefault: true,
             },
+            {
+                href: `/projects/${PROJECT}/features/demoApp.step4`,
```

Comment:
Is it ok having step4 twice?

Reply:
Should be safe, as this only triggers a navigation if the current URL does not match, but I can double check just in case.